### PR TITLE
Update publications.md

### PR DESCRIPTION
**Pull request by mforets**

### CODE REVIEW

File: docs/src/publications.md [hunk 1]
```diff
@@ -6,6 +6,26 @@ Pages = ["publications.md"]
...
+
+- **ARCH-COMP19 Category Report: Continuous and Hybrid Systems with Nonlinear Dynamics.** Fabian Immler, Matthias Althoff, Luis Benet, Alexandre Chapoutot, Xin Chen, Marcelo Forets, Luca Geretti, Niklas Kochdumper, David P. Sanders and Christian Schilling (2019) ARCH19. 6th International Workshop on Applied Verification of Continuous and Hybrid Systems, vol 61, pages 41--61 [doi: 10.29007/bj1w](https://easychair.org/publications/paper/1gbP). Packages: [Reachability.jl](https://github.com/JuliaReach/Reachability.jl).
+
+**Abstract.** *We present the results of a friendly competition for formal verification of continuous and hybrid systems with nonlinear continuous dynamics. The friendly competition took place as part of the workshop Applied Verification for Continuous and Hybrid Systems (ARCH) in 2019. In this year, 6 tools Ariadne, CORA, DynIbex, Flow*, Isabelle/HOL, and JuliaReach (in alphabetic order) participated. They are applied to solve reachability analysis problems on four benchmark problems, one of them with hybrid dynamics. We do not rank the tools based on the results, but show the current status and discover the potential advantages of different tools.*
```
Comment (schillic) [inline docs/src/publications.md]: The `*` in `Flow*` destroys the italics mode.
```suggestion
**Abstract.** _We present the results of a friendly competition for formal verification of continuous and hybrid systems with nonlinear continuous dynamics. The friendly competition took place as part of the workshop Applied Verification for Continuous and Hybrid Systems (ARCH) in 2019. In this year, 6 tools Ariadne, CORA, DynIbex, Flow*, Isabelle/HOL, and JuliaReach (in alphabetic order) participated. They are applied to solve reachability analysis problems on four benchmark problems, one of them with hybrid dynamics. We do not rank the tools based on the results, but show the current status and discover the potential advantages of different tools._
```

File: docs/src/publications.md [hunk 2]
```diff
@@ -6,6 +6,26 @@ Pages = ["publications.md"]
 
 This page lists publications about the JuliaReach ecosystem and its applications.
 
+## ARCH 2019 Competition NLN Category Report
+
+- **ARCH-COMP19 Category Report: Continuous and Hybrid Systems with Nonlinear Dynamics.** Fabian Immler, Matthias Althoff, Luis Benet, Alexandre Chapoutot, Xin Chen, Marcelo Forets, Luca Geretti, Niklas Kochdumper, David P. Sanders and Christian Schilling (2019) ARCH19. 6th International Workshop on Applied Verification of Continuous and Hybrid Systems, vol 61, pages 41--61 [doi: 10.29007/bj1w](https://easychair.org/publications/paper/1gbP). Packages: [Reachability.jl](https://github.com/JuliaReach/Reachability.jl).
```
Comment (schillic) [inline docs/src/publications.md]: Should we mention the cooperation with the `Taylor*` packages?

Comment (mforets) [reply docs/src/publications.md]: This is a good idea. Actually we can add the paragraph describing JuliaReach here.

Comment (schillic) [reply docs/src/publications.md]: Good, we can add explicit links as well.

**Pull request merged**